packages/core/admin/admin/src/components/Roles/ConditionsModal/ActionRow/index.js

```diff
@@ -19,50 +23,27 @@ const ActionRow = ({
   const { formatMessage } = useIntl();
 
   return (
-    <Wrapper isGrey={isGrey}>
-      <Padded style={{ width: 200 }} top left right bottom size="sm">
-        <Flex>
-          <Text
-            lineHeight="19px"
-            color="grey"
-            fontSize="xs"
-            fontWeight="bold"
-            textTransform="uppercase"
-          >
-            {formatMessage({
-              id: 'Settings.permissions.conditions.can',
-            })}
-            &nbsp;
-          </Text>
-          <Text
-            title={label}
-            lineHeight="19px"
-            fontWeight="bold"
-            fontSize="xs"
-            textTransform="uppercase"
-            color="mediumBlue"
-            style={{ maxWidth: '60%' }}
-            ellipsis
-          >
-            {formatMessage({
-              id: `Settings.roles.form.permissions.${label.toLowerCase()}`,
-              defaultMessage: label,
-            })}
-          </Text>
-          <Text
-            lineHeight="19px"
-            color="grey"
-            fontSize="xs"
-            fontWeight="bold"
-            textTransform="uppercase"
-          >
-            &nbsp;
-            {formatMessage({
-              id: 'Settings.permissions.conditions.when',
-            })}
-          </Text>
-        </Flex>
-      </Padded>
+    <RowWrapper background={isGrey ? 'neutral100' : 'neutral0'}>
+      <Row paddingLeft={6} style={{ width: 180 }}>
+        <TableLabel textColor="neutral500">
+          {formatMessage({
+            id: 'Settings.permissions.conditions.can',
+          })}
+          &nbsp;
+        </TableLabel>
+        <TableLabel title={label} textColor="primary600">
+          {formatMessage({
+            id: `Settings.roles.form.permissions.${label.toLowerCase()}`,
+            defaultMessage: label,
+          })}
+        </TableLabel>
+        <TableLabel textColor="neutral500">
+          &nbsp;
+          {formatMessage({
+            id: 'Settings.permissions.conditions.when',
+          })}
+        </TableLabel>
+      </Row>
       <ConditionsSelect
```

Comment:
You might need to migrate this component to our new DS.

Reply:
I did it. What did I do wrong ?